content/2_Bronze/Intro_Sorting.mdx

```diff
@@ -161,21 +157,19 @@ $\Theta(N^2)$ behavior in quicksort.
 See [here](https://codeforces.com/contest/1324/hacks/625031/) for an example of
 a solution that was hacked on CodeForces.
 
-Two ways to avoid this:
+There's two ways to avoid this:
```

Comment:
There are?

Reply:
that was what the og said
i'm just changing the grammar

Reply:
"There's two ways" is incorrect

Reply:
```suggestion
There are two ways
```